proteus/LinearSolvers.py

```diff
@@ -2465,3 +2466,142 @@ def _createOperator(self,coeff,matrixDict,A):
                                                                           self.model.csrColumnOffsets[(ci,cj)],
                                                                           matrixDict[ci][cj],
                                                                           A)
+
+
+class IterativeMethod:
```

Comment:
What's the difference in an iterative method and a LinearSolver or  a Preconditioner? Just don't want to proliferate distinct class hierarchies without a good reason.